test/containers/run-tests

```diff
@@ -22,17 +22,11 @@ import imp
 import os
 import string
 import unittest
+from parent import TEST_DIR, BASE_DIR
+import testlib
 
-sys.dont_write_bytecode = True
-
-base_dir = os.path.dirname(os.path.realpath(__file__))
-testdir = os.path.dirname(base_dir)
-sys.path.append(testdir)
 
-from verify import parent
-parent  # pyflakes
-
-from common import testlib
+sys.dont_write_bytecode = True
```

Comment:
Let's drop that as well for consistency, or keep it in urls-check as well? (But of course better if we can just drop it)

Reply:
I left this one (along with `test/common/run-tests`) because they import the individual testcases, which puts them in a bit of a different category.

Reply:
Ack.